com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Math/Interpolation/LerpNode.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class LerpNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,      // Version
+            "Lerp", // Name
+            "Out = lerp(A, B, T);",
+            new ParameterDescriptor("A", TYPE.Any, Usage.In),
+            new ParameterDescriptor("B", TYPE.Any, Usage.In),
```

Comment:
In the old SG, B has a default value float 1.

Reply:
Added the missing default value.